parser/v2/textparser.go

```diff
@@ -18,5 +20,22 @@ var textParser = parse.Func(func(pi *parse.Input) (t Text, ok bool, err error) {
 		return
 	}
 
+	t.LineBreak = true
+
+	v, ok := pi.Peek(maxIgnoreNewLineTagLength + 3)
+
+	if ok {
+		for _, i := range ignoreNewLinesTags {
+			if strings.Contains(v, "<"+i+">") || strings.Contains(v, "</"+i+">") {
+				t.LineBreak = false
+				break
+			}
+		}
+	}
+
+	if line, _ := pi.Peek(1); line == "\n" {
+		t.LineBreak = true
+	}
```

Comment:
I'm not convinced by this section.

This looks like it's peeking forward into potentially non text code to look for elements.

Reply:
Yes by default the parser assume that every line after text should got a new line.

So here I have try some rule to decide if this text must wrap or not after.

But now I re read my code I see some behavior that are useless.

Text should always wrap except if after it there is an inline tag.

This rule can just be made in the `types.go` file when we reconstruct the code.

I will try so the optimize this and so try to just don't touch this file.